docs/auth/atlassian/provider.md

```diff
@@ -46,16 +46,16 @@ auth:
       development:
         clientId: ${AUTH_ATLASSIAN_CLIENT_ID}
         clientSecret: ${AUTH_ATLASSIAN_CLIENT_SECRET}
-        scopes: ${AUTH_ATLASSIAN_SCOPES}
+        scope: ${AUTH_ATLASSIAN_SCOPES}
```

Comment:
Not opposed to changing this back to `scopes`, but `scope` is consistent with terminology elsewhere.

Reply:
That's fine, but please make the code read both the old and the new name, so you don't break existing users.

Like `config.getOptionalString('scope') ?? config.getOptionalString('scopes') ?? someDefaultValue` or so

Reply:
Ok, updated.